src/client/opamConfigCommand.ml

```diff
@@ -21,39 +21,36 @@ let list t ns =
   log "config-list";
   if ns = [] then () else
   let list_vars name =
-    if OpamPackage.Name.to_string name = "-" then
-      let conf = t.switch_config in
-      List.map (fun (v,c) ->
-          OpamVariable.Full.global v,
-          OpamVariable.string_of_variable_contents c,
-          "")
-        (conf.OpamFile.Switch_config.variables)
-    else
```

Comment:
Indeed this code never worked. We should probably also remove it from the documentation: https://github.com/ocaml/opam/commit/56e3e6c62e8178d3634c3f083f8b71393340d8e9#diff-e674376a5ba688da9b6f11571be146af2e7bbde7c583630c6e8d4fc15c575035R1013

Reply:
updated